### DISCUSSION TITLE
Jump to end of page?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#3386] [Q&A]
This may be the dimmest of dim questions, but is there any way to jump to the end of a page in Xournalpp? I can find no combination of keystrokes that does this. 

I'm using the first hotfix release on Windows 10.

**Comment by Technius** [comment]
No worries, no question is dim. I don't think there's such a hotkey right now; a workaround is to jump to the next page and then scroll up. It would be pretty easy to implement though (please submit a feature request).

**Reply by NelkonParker** [reply]
Sorry, I've just seen this, though I note that it was posted 22 days ago - slightly odd as I looked last week and saw no response.

Anyway, sure, I'll submit a feature request - where do I do that though? On the "issues" page?

**Reply by rolandlo** [reply]
Yes, on the `Issues` page you can open a new Issue (green button) and use the `Feature Request` template.
